submissions/lukmanokunade/HTML/Registration.html

```diff
@@ -2,21 +2,41 @@
 <html>
     <head>
         <title>Deuces</title>
+        <link href="style.css" rel="stylesheet">
     </head>
+
     <body>
-        <h1>Description</h1>
-        <p>If you would like to experience the best of online shopping for men, women and kids, you are<br> at the right place. Deuces is the ultimate destination for fashion and lifestyle,<br> being host to a wide array of merchandise including clothing, footwear, accessories, jewellery,<br> personal care products and more. It is time to redefine your style statement with our treasure-trove of trendy items. <br>Our online store brings you the latest in designer products straight out of fashion houses. <br>You can shop online at Deuces from the comfort of your home and get your favourites delivered right to your doorstep.</p>
+    <h1>Description</h1>
```

Comment:
Please fix all indentations and we are good to go. Check Styleguide.md in the repository

Reply:
I have been able to properly fix the indentations